bedrock/firefox/templates/firefox/hello/index-2016-b.html

```diff
@@ -0,0 +1,22 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/hello/index-2016.html" %}
```

Comment:
Confirmed that we won't worry about testing on the Hello page until March 8, so we'll activate the optimizely snippet's waffle switch at the same time we activate the page's waffle switch.